README.md

```diff
@@ -15,3 +15,4 @@ Do you want to help organizations see the light around how to use SBOMs (and oth
 | ------------------------------------ | ---------------------------------------- | ------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | ----------------------------------------------------------------------------------------------------------------------------------------- |
 | *EXAMPLE: ACME Inc.*                 | *SBOM*                                   | *Emergency Vulnerability Incident Response* | *ACME Inc. used their vendors' SaaSBOMs to identify and contact vendors vulnerable to Log4Shell. Most vendors had already begun remediating, but a few vendors triggered their incident response process \*because\* ACME had contacted them quickly*                                                                                                                                                              | [*\[BSidesFoobar 2023\] Finally: a real example of SBOMs providing tangible security value*](https://www.youtube.com/watch?v=FAKEVIDEOID) |
 | *EXAMPLE:  Fortune 500 Bank*         | *SaaSBOM*                                | *New Vendor Security Risk Assessments*      | *A Fortune 500 bank began requiring SaaSBOM's from their critical SaaS application providers that process regulated data (PII, PHI, PCI, etc.). Because of the added visibility into their SaaS vendors' software currency (or lack thereof), they were able to deny procurement requests for 10 vendors that would have otherwise been approved while helping the business find alternative, more secure vendors* | [*Trust us: a real and honest customer case study for how to get practical value from SaaSBOMs (from SBOMify Inc.)*](https://example.com) |
+| Deliveroo                            | SBOM                                     | Operational impact                          | As part of triaging an operational incident, engineers at Deliveroo discovered that a number of services were affected by a race condition in an internal dependency, which could lead to severe operational impact. Using the known dependency data, and understanding of the business criticality of repos affected, the teams could prioritise work accordingly.                                                | [Dependency Management Data case study: Deliveroo and a potential race condition with a Kafka sidecar](https://dmd.tanna.dev/case-studies/deliveroo-kafka-sidecar/)                      |
```

Comment:
This wasn't technically an SBOM, but a list of the dependencies the org understands - worth using something else here?